cmd/memresolverconfigbuilder_test.go

```diff
@@ -130,20 +140,33 @@ func Test_MemResolverErrorBadClaim(t *testing.T) {
 	require.Error(t, builder.Add([]byte("bad")))
 }
 
-func Test_MemResolverQuotesRelativePath(t *testing.T) {
-	builder := NewMemResolverConfigBuilder()
-	d, err := builder.Generate("./test/relative.jwt")
+func Test_MemResolverDir(t *testing.T) {
+	ts := NewTestStore(t, "O")
+	defer ts.Done(t)
+	ts.AddAccount(t, "A")
+	ts.AddAccount(t, "B")
+
+	out := filepath.Join(ts.Dir, "conf")
+	_, _, err := ExecuteCmd(createServerConfigCmd(), "--mem-resolver",
+		"--dir", out)
```

Comment:
I wonder if there is a reasonable way to test the -- option, which i assume sends to std out.

Reply:
added one